packages/devtools_app/lib/src/screens/memory/memory_analyzer.dart

```diff
@@ -448,14 +451,14 @@ bool? displayObject(
   createTreeNodes = false,
 }) {
   if (depth >= maxDepth) return null;
-  if (live.references!.isEmpty) return true;
+  if (live.references?.isEmpty != false) return true;
```

Comment:
nit: I think `live.references?.isEmpty ?? true` is a bit more readable and consistent with the rest of the codebase

Reply:
done